File: fizzbuzz.swift
var number = 0
let finish = 20

while number < finish {
  number = number + 1

  if (number % 3 == 0 && number % 5 == 0) {
    print("FizzBuzz!")
  } else if (number % 3 == 0) {
    print("Fizz!")
  } else if (number % 5 == 0 ) {
    print("Buzz!")
  } else {
    print(number)
  }
}

File: fizzbuzz.rb
number = 0
finish = 20

while number < finish
  number +=1

  if number % 3 == 0 and number % 5 == 0
    puts "FizzBuzz!"
  elsif number % 3 == 0
    puts "Fizz!"
  elsif number % 5 == 0
    puts "Buzz!"
  else
    puts number
  end
end
